internal/ackhandler/received_packet_history.go

```diff
@@ -22,25 +22,26 @@ func newReceivedPacketHistory() *receivedPacketHistory {
 }
 
 // ReceivedPacket registers a packet with PacketNumber p and updates the ranges
-func (h *receivedPacketHistory) ReceivedPacket(p protocol.PacketNumber) {
+func (h *receivedPacketHistory) ReceivedPacket(p protocol.PacketNumber) bool /* is a new packet (and not a duplicate / delayed packet) */ {
```

Comment:
Maybe return an enum instead? Bools as return values can be hard to understand when the function name doesn't make it clear.

Reply:
How would that work? There are no enums in Go. Do you want to define a type alias for a bool?

```go
type isNewType bool

const (
    isNew isNewType = true
    isNotNew isNewType = false
)
```